src/jobs/go-lint.yaml

```diff
@@ -0,0 +1,47 @@
+description: "Runs golanglint-ci against the codebase."
+parameters:
+  disable:
+    default: ""
+    description: "The names of default linting modules to skip."
+    type: "string"
+  disable-all:
+    default: false
+    description: "If set, will disable all linters (except those included by '--enable')."
+    type: "boolean"
+  enable:
+    default: ""
+    description: "The names of additional linting modules to run."
+    type: "string"
+  new-from-rev:
+    default: ""
+    description: "Show only new issues created after git revision REV."
+    type: "string"
+  presets:
+    default: "none"
```

Comment:
Maybe I misunderstand this option, but don't we always want to have at least one?

Reply:
There is a default suite, this is if you only want a certain block of linters (these run some which aren't included in the default as well)